yaml/pois.yaml

```diff
@@ -1,16 +1,424 @@
+           WHEN way_area >=  100000 THEN 13
+           WHEN way_area >=   50000 THEN 14
+           WHEN way_area >=    2000 THEN 15
+           ELSE                          16
+      END
+  - &tier5_min_zoom |
+      CASE WHEN way_area >= 1000000 THEN 10
+           WHEN way_area >=  400000 THEN 11
+           WHEN way_area >=  200000 THEN 12
+           WHEN way_area >=   50000 THEN 13
+           WHEN way_area >=   20000 THEN 14
+           WHEN way_area >=    2000 THEN 15
+           ELSE                          16
+      END
+  - &tier6_min_zoom |
+      CASE WHEN way_area >= 500000 THEN 12
+           WHEN way_area >= 100000 THEN 13
+           WHEN way_area >=  50000 THEN 14
+           WHEN way_area >=   5000 THEN 15
+           ELSE                         16
+      END
+  - &us_forest_service
+        - United States Forest Service
+        - US Forest Service
+        - U.S. Forest Service
+        - USDA Forest Service
+        - United States Department of Agriculture
+        - US National Forest Service
+        - United State Forest Service
+        - U.S. National Forest Service
+  - &us_parks_service
+        - United States National Park Service
+        - National Park Service
+        - US National Park Service
+        - U.S. National Park Service
+        - US National Park service
 filters:
   - filter: {disused: -no}
     min_zoom: null
     output:
       kind: {expr: null}
+
+  ############################################################
+  # NOT IN ANY TIER
+  #
+  # These depend on being run before the rule for tourism=attraction,
+  # otherwise that rule will trigger first. So these have been moved
+  # up here.
+  ############################################################
+  - filter: {man_made: windmill}
+    min_zoom: CASE WHEN tags ? 'tourism' AND tags->'tourism' = 'attraction' THEN 14
+      ELSE 15 END
+    output: {kind: windmill}
+  - filter: {man_made: lighthouse}
+    min_zoom: CASE WHEN tags ? 'tourism' AND tags->'tourism' = 'attraction' THEN 14
+      ELSE 15 END
+    output: {kind: lighthouse}
+  - filter: {man_made: observatory}
+    min_zoom: 15
+    output: {kind: observatory}
+
+  ############################################################
+  # TIER 1
+  ############################################################
+  - filter:
+      historic: battlefield
+      not: {operator: *us_forest_service}
+    min_zoom: { min: [ { max: [ 10, { lit: zoom + 4 }, { lit: *tier1_min_zoom } ] }, 17 ] }
+    output: {kind: battlefield, tier: 1}
+  - filter:
+      boundary: national_park
+      not: {operator: *us_forest_service}
+    min_zoom: { min: [ { max: [ { lit: zoom + 3.5 }, { lit: *tier1_min_zoom } ] }, 10 ] }
+    output:
+      kind: national_park
+      tier: 1
+
+  ############################################################
+  # TIER 2
+  ############################################################
+  - filter:
+      boundary: national_park
+      not: {operator: *us_parks_service}
```

Comment:
Agree this basic `national_park` logic is fine for this PR, to be refined in https://github.com/tilezen/vector-datasource/issues/987.